File: blog_backend/littleseventhirty-blog/src/main/java/cn/edu/tjufe/zql/domain/entity/Tag.java
package cn.edu.tjufe.zql.domain.entity;

import cn.edu.tjufe.zql.domain.ViewObjectConvertible;
import com.baomidou.mybatisplus.annotation.FieldFill;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * @author: littleseventhirty
 * @description: 标签实体类
 * @date: 2026/1/28-16:42
 **/
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@TableName("t_tag")
public class Tag implements ViewObjectConvertible {
    // 标签id
    private Long tagId;
    // 标签名称
    private String tagName;
    // 创建时间
    @TableField(fill = FieldFill.INSERT)
    private Date createTime;
    // 更新时间
    @TableField(fill = FieldFill.INSERT_UPDATE)
    private Date updateTime;
    // 是否删除
    private Integer isDelete;
}

File: blog_frontend/littleseventhirty-blog/src/mock/tag.ts
import Mock from "mockjs";

Mock.mock("/api/tag/list", "get", {
  code: 200,
  message: "success",
  "data|15-25": [
    {
      // 生成一个随机的整数 ID
      "tagId|+1": 1,
      // 生成一个随机的标签名称（中文词）
      tagName: "@CTITLE(2, 4)",
      // 生成文章数量，范围在 0 到 100 之间
      "articleCount|0-100": 1,
      // 生成创建时间，随机在过去的一年内
      createTime: "@DATETIME",
      // 生成更新时间，随机在过去的一周内
      updateTime: "@DATETIME"
    }
  ]
});